lib/external_asset_pipeline/manifest.rb

```diff
@@ -29,6 +29,8 @@ def data
 
     def load
       JSON.parse(@config.manifest_path.read)
+    rescue Errno::ENOENT
+      {}
```

Comment:
We have this warning when a `ServerManifest` is used but the corresponding server isn't `running?`:
https://github.com/rails-front-end/external_asset_pipeline/blob/2bbf45e615e48dc8afd5fbcb7b0f0bce0c761633/lib/external_asset_pipeline/server_manifest.rb#L27-L29

Do you think it's worth logging a similar warning in this scenario?

Reply:
Alternatively, we could add a `Configuration` attribute that defaults to `true` but can be set to `false` in the development environment, and this code could re-`raise` the exception if that attribute is `true`

Reply:
Done!